SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -42,6 +42,8 @@ ASpatialFunctionalTest::ASpatialFunctionalTest()
 	bAlwaysRelevant = true;
 
 	PrimaryActorTick.TickInterval = 0.0f;
+
+	PreparationTimeLimit = 30.0f;
```

Comment:
Would suggest appending the unit e.g. PreparationTimeLimitInSeconds

Reply:
Unfortunately we inherit this variable all the way from AFunctionalTest, which is Epic-written and lives in the engine :(.
So while I agree, probably won't be done here.

Reply:
fair enough would insert a comment
`// in seconds`